docs/codeql/codeql-cli/creating-and-working-with-codeql-packs.rst

```diff
@@ -55,7 +57,7 @@ You can add dependencies on CodeQL packs using the command ``codeql pack add``.
 
 The version number is optional. If you leave off the version number, the latest version will be added.
```

Comment:
```suggestion
The version range is optional. If you leave off the version range, the latest version will be added. Otherwise, the latest version that satisfies the requested range will be added.
```